protobuf/gloria/gloria.proto

```diff
@@ -124,4 +124,6 @@ message Download {
 message Distribution {
     // Set true if data should be sent to the IGS
     bool igs = 1;
+    // Set true if data should be sent to the LINZ
```

Comment:
This line isn't quite true, we're not planning to send data to LINZ.
Should maybe say "// Set true if this site belongs to LINZ"